misc/python/materialize/mzcompose/services.py

```diff
@@ -65,7 +65,8 @@ def __init__(
         restart: Optional[str] = None,
         use_default_volumes: bool = True,
         ports: Optional[List[str]] = None,
-        bootstrap_system_parameters: Optional[List[str]] = None,
+        cli_system_variable_defaults: Optional[List[str]] = None,
+        additional_cli_system_variable_defaults: Optional[List[str]] = None,
```

Comment:
`extra` is our convention for mzcompose service parameters like this.

```suggestion
        extra_cli_system_variable_defaults: Optional[List[str]] = None,
```